aten/src/ATen/native/mps/operations/LossOps.mm

```diff
@@ -995,15 +995,17 @@ Tensor huber_loss_mps(const Tensor& input, const Tensor& target, int64_t reducti
 
 // MSELoss
 TORCH_IMPL_FUNC(mse_loss_out_mps)(const Tensor& input, const Tensor& target, int64_t reduction, const Tensor& output_) {
-  string op_name = __func__;
```

Comment:
why this change?

Reply:
See PR description, for `TORCH_IMPL_FUNC` macro `__func__` will always return `impl`, which is bad for MPSGraph cache keys as well as for error messages.